packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -2064,6 +2225,8 @@ class MultiTypeaheadSelectInputCustomObjects extends React.Component {
 
 ### Plain multiple typeahead
 
+To make a typeahead that allows users to make multiple selections, pass “variant=typeaheadMulti” into the `<Select>` component. The following example also styles the select list plainly by using the `isPlain` property.  
```

Comment:
This example seems like it's more a duplicate of the "Multiple select" example, except `isPlain` is passed in. @mcarrano I feel like one of the examples could be removed, and the one that remains could just have a checkbox to enable/disabled `isPlain`